osu.Game/Skinning/LegacySkinTransformer.cs

```diff
@@ -23,6 +24,18 @@ protected LegacySkinTransformer(ISkin skin)
         {
         }
 
+        public override Texture? GetTexture(string componentName, WrapMode wrapModeS, WrapMode wrapModeT)
+        {
+            switch (componentName)
+            {
+                case "fountain-star":
+                    componentName = "star2";
+                    break;
```

Comment:
As a last check I decided to see if this does anything (because I kinda glossed over this before), and a possible problem here is that if this was intended to permit skinnability in main menu too rather than just in gameplay, then it doesn't work:

https://github.com/ppy/osu/assets/20418176/ddac907c-27de-4cd1-82e3-17e08e386e6c

Minimal skin used to reproduce: [star2.osk.zip](https://github.com/ppy/osu/files/12131610/star2.osk.zip)

Note that the custom `star2` sprite works in gameplay and not in main menu, and that is because skin transformers are actually not constructed outside of gameplay at all.

Reply:
Right, I moved this out of the `LegacySkin` class as it felt better, but alas...

Maybe transformers should have `Gameplay` in their name.

Reply:
Wouldn't be opposed to such a rename (in another pull, which probably goes without saying).

Reply:
Have moved it to a better place, works now:

![image](https://github.com/ppy/osu/assets/191335/396f1bc9-3507-4040-9f5f-d89bb4587adc)